micro_emulation_plans/src/user_execution/dot_one/README.md

```diff
@@ -0,0 +1,42 @@
+# README for User Execution of OneNote File (.one)
+
+## Prerequisites
+
+- OS: Built and tested on Windows 10
+- Application: Microsoft OneNote
+
+## Executing the Application
+
+- The tool distribution contains a zip file containing the `Dot_One.exe` file.
+- Double-click `Dot_One.exe`, which will:
+
+1. Create an HTTP server, which acts as adversary infrastructure
+2. Prompt user with `press enter to exit...` in a cmd window
+3. Drop malicious `.one` file to disk and opens in OneNote
+4. User is prompted to double-click the `Accept License` button, which serves as our phishing object
+5. Once the button is pressed, a batch script runs in the background creating a popup
+6. Embedded batch script queries fileserver for encoded base64 powershell command from file `README.md` to create a benign scheduled task for persistence
+7. Pressing enter on the original cmd window will cleanup all files, close the httpserver, and delete the scheduled task
+
+## Customizing the PowerShell Command
+
+### Prerequisites
+
+- Application: Microsoft Visual Studio
+
+1. Use [this cyberchef link](https://gchq.github.io/CyberChef/#recipe=Encode_text('UTF-16LE%20(1200)')To_Base64('A-Za-z0-9%2B/%3D')&input=c2NodGFza3MgL0NyZWF0ZSAvRiAvU0MgTUlOVVRFIC9NTyAzIC9TVCAwNzowMCAvVE4gQ01EVGVzdFRhc2sgL1RSICJjbWQgL2MgZGF0ZSAvVCA%2BIEM6XFdpbmRvd3NcVGVtcFxjdXJyZW50X2RhdGUudHh0Ig), replacing the `input` with the desired PowerShell commands
+2. Paste the `output` from CyberChef into a new file `README.md`
```

Comment:
Double checking the instructions here. A "new" README.md will overwrite the existing one. Might be easier to explain that you edit README.md and replace the contents with the new b64 string?

Also is it really necessary (I'm not sure how VS handles resources) to edit Resource1.resx if the new README has the same name as the oldone?

Reply:
Fixed phrasing to make a little more clear. I haven't tested just replacing the file or overwriting it in the folder and then building but afaik that should not post any issues